packages/test/test-end-to-end-tests/src/test/data-virtualization/groupIdInSummary.spec.ts

```diff
@@ -37,6 +41,10 @@ const interceptResult = <T>(
 	return fn;
 };
 
+const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderBase => ({
```

Comment:
I think you can still use `createTestConfigProvider`

There's a `.set("your-flag", yourValue)` on it

Reply:
Not sure, but that was not working properly as per my debugging. But I might be wrong. As this is definitely working then we can keep this here.